diesel_cli/tests/print_schema.rs

```diff
@@ -229,6 +229,12 @@ fn print_schema_sqlite_implicit_foreign_key_reference() {
     test_print_schema("print_schema_sqlite_implicit_foreign_key_reference", vec![]);
 }
 
+#[test]
+#[cfg(feature = "postgres")]
+fn print_schema_respect_type_name_case() {
+    test_print_schema("print_schema_respect_type_name_case", vec!["--with-docs"])
```

Comment:
```suggestion
fn print_schema_respects_type_name_case() {
    test_print_schema("print_schema_respects_type_name_case", vec!["--with-docs"])
```

and of course corresponding file names